node_modules/oae-core/userdetails/js/userdetails.js

```diff
@@ -0,0 +1,181 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['jquery', 'oae.core'], function($, oae) {
+
+    return function(uid) {
+
+        // The widget container
+        var $rootel = $('#' + uid);
+
+        /**
+         * Disable any input fields or buttons in the user details form
+         */
+        var disableUserDetailsForm = function() {
+            $('#userdetails-form input, #userdetails-form button', $rootel).prop('disabled', true);
```

Comment:
Sorry about this, but it seems like the UI actually disables with `#userdetails-form *` which I wouldn't have expected.

Also, my recommendation earlier about avoiding duplication wasn't to extract a method out of enabling and disabling, but rather hold `$('#userdetails-form input, #userdetails-form button', $rootel)` into a variable so that we don't have to copy/paste the selector into 2 different spots. But we're still doing that here.